server/tests/src/test/java/org/infinispan/server/persistence/CustomNonBlockingStore.java

```diff
@@ -23,7 +23,7 @@
    public CompletionStage<Void> start(InitializationContext ctx) {
       marshallableEntryFactory = ctx.getMarshallableEntryFactory();
       AdvancedCache<String, String> object = (AdvancedCache) ctx.getCache().getAdvancedCache()
-            .withMediaType(MediaType.APPLICATION_OBJECT_TYPE, MediaType.APPLICATION_OBJECT_TYPE);
+            .withMediaType(MediaType.APPLICATION_OBJECT, MediaType.APPLICATION_OBJECT);
```

Comment:
You missed a cast that's not needed any more on line 25

Reply:
removed